backend/src/test/java/com/cruru/question/service/QuestionServiceTest.java

```diff
@@ -82,4 +86,53 @@ void create() {
         assertThat(questions).hasSize(1);
         assertThat(questions.get(0)).isEqualTo(question);
     }
+
+    @DisplayName("Question 엔티티의 정보를 이용하여 Response DTO로 변경한다.")
+    @ParameterizedTest(name = "{index} - {1}")
```

Comment:
```suggestion
    @ParameterizedTest(name = "Expected : {0}, Actual : {1}")
```

이런식으로 구분하고 사용되지 않는 index는 제거하는 것이 어떨까요? 마찬가지로 provideQuestionsAndResponses에서는 반환값에 index를 담아주지 않아도 될 것 같습니다.

Reply:
좋은 방법인 것 같습니다!
점심 마치면 바로 적용해볼게요 :)